libs/common/files.hpp

```diff
@@ -26,11 +27,19 @@ namespace iroha {
                            const logger::LoggerPtr &log);
 
   /**
-   * Read file, and either return its contents as a string
+   * Read file in text mode, and either return its contents as a string
    * or return the error as a string
    * @param path - path to the file
    */
-  iroha::expected::Result<std::string, std::string> readFile(
+  iroha::expected::Result<std::string, std::string> readTextFile(
+      const std::string &path);
```

Comment:
that adds no benefits as this file is not included transitively.
but it would add difficulties, as everywhere we use it, we will need to include the definition as well